stdlib/stdlib.mli

```diff
@@ -872,6 +872,11 @@ val output_substring : out_channel -> string -> int -> int -> unit
    a byte sequence.
    @since 4.02.0 *)
 
+val output_bigstring : out_channel -> bigstring -> int -> int -> unit
+(** Same as {!output}, but takes the data from a big string.
+
+    @since 4.08.0 *)
```

Comment:
Give that it's easier to add new modules to the stdlib now, should we consider not polluting the global namespace with "IO" functions today, and instead add them to a separate module? (possibly later and outside of this MR)

Reply:
Yes, we should consider it. However:

- I don't see anyone working on this hypothetical `IO` module.
- If and when such a module is introduced we will need to deprecate existing I/O functions in `Stdlib` anyway; adding one more function will not change anything much.
- Personally, I am not super confident that such a module will land soon because, contrary to all the recently considered new modules (`Float`, `Seq`, `Option`, `Result`), the design space seems much larger and closely related to things such as effects and multicore, which will complicate matters.

Having said that, of course I am open to discussion, and if anyone is sitting on a `IO` proposal, I am happy to delay the inclusion of this function.

Reply:
> the design space seems much larger and closely related to things such as effects and multicore

We are indeed working on an effects-based IO library that supports direct-style function calls using effects.  It's probably not worth doing a large stdlib rearrangement before that prototype is complete.

Reply:
Well, conveniently this just came up: https://github.com/ocaml/ocaml/pull/640#issuecomment-411427218. And I think this is another point in favour of not adding the IO functions in Stdlib:

>>  One way of accomplishing that is to directly expose the equality between bigstring and Bigarray.Array1.t
>
> That was my initial intention, but it complicates the definition of functions on this type in Stdlib/Pervasives (e.g. output_bigstring, etc).